addons/medical_treatment/functions/fnc_treatmentBandage.sqf

```diff
@@ -8,21 +8,20 @@
  * 2: Body part <STRING>
  * 3: Treatment class name <STRING>
  * 4: Item <STRING>
- * 5: specific Spot <NUMBER> (default: -1)
  *
  * Return Value:
  * Succesful treatment started <BOOL>
  *
- * Public: Yes
+ * Public: No
  */
 #include "script_component.hpp"
 
-params ["_caller", "_target", "_bodyPart", "_className", "_items", "", ["_specificSpot", -1]];
```

Comment:
Same here, removing ` ["_specificSpot", -1]` means we are dropping support for selecting the injury to bandage in the UI.